crates/bevy_asset/src/io/wasm_asset_io.rs

```diff
@@ -52,6 +52,7 @@ impl AssetIo for WasmAssetIo {
         &self,
         _path: &Path,
     ) -> Result<Box<dyn Iterator<Item = PathBuf>>, AssetIoError> {
+        bevy_log::warn!("Loading folders is not supported in WASM");
```

Comment:
To keep in line with the name of the method
```suggestion
        bevy_log::warn!("Reading a directory is not supported in WASM");
```

Reply:
I thought that this phrasing would be more useful to normal users who would see this message after attempting to use [`load_folder`](https://docs.rs/bevy_asset/latest/bevy_asset/struct.AssetServer.html#method.load_folder).

Reply:
Right, I guess that makes sense then since it's probably going to be the more common use case